src/pip/_internal/vcs/subversion.py

```diff
@@ -53,20 +69,63 @@ def get_vcs_version(self):
         version_prefix = 'svn, version '
         version = self.run_command(['--version'], show_stdout=False)
         if not version.startswith(version_prefix):
-            return None
+            return ()
 
         version = version[len(version_prefix):].split()[0]
         version_list = version.split('.')
         try:
             parsed_version = tuple(map(int, version_list))
         except ValueError:
-            return None
-
-        if not parsed_version:
-            return None
+            return ()
 
         return parsed_version
 
+    def get_vcs_version(self):
```

Comment:
Since this is the primary method that should be used, this should have a docstring like `call_vcs_version()` does. And if that's done, then `call_vcs_version()`'s docstring should probably be rephrased to say something like, "Query the version from the vcs command," to contrast with this method which incorporates the caching.